extensions/msal-node-extensions/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Authentication Extensions for Node
+The Microsoft Authentication Extensions for Node offers secure mechanisms for client applications to perform cross-platform token cache serialization and persistence. It gives additional support to the Microsoft Authentication Library for Node (MSAL).
+
+MSAL Node supports an in-memory cache by default and provides the ICacheManager to perform cache serialization. Developers are required to implement their own cache persistance across multiple platforms and Microsoft Authentication Extensions makes this simpler.
+
+Supported platforms are Windows, Mac and Linux:
+
+- Windows - DPAPI is used for encryption.
+- MAC - The MAC KeyChain is used.
+- Linux - LibSecret is used for encryption.
```

Comment:
make it a table with `Platform` and `Description` as fields?

Reply:
I don't think theres a need for a table here, bullet points work. Can always add it later if more stuff is added.